db/migrate/20231110010253_workflow_execution.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# Migration to add WorkflowExecution table
+class WorkflowExecution < ActiveRecord::Migration[7.1]
+  def change # rubocop:disable Metrics/MethodLength
+    create_table :workflow_executions do |t|
+      t.jsonb :metadata, null: false, default: WorkflowMetadata.dump(
+        WorkflowMetadata.new(workflow_name: '', workflow_version: '')
+      )
+      t.jsonb :workflow_params
```

Comment:
Can you add a default value of an empty hash, see migrations for Attachment or Sample where `metadata` column is defined.

Reply:
fixed in https://github.com/phac-nml/irida-next/pull/308/commits/b93ceec0d2aa3386add4aafbf34c30d56610ee7f